README.md

```diff
@@ -1,29 +1,288 @@
+    - Functional
+    - System inputs and outputs
+    - Application programming interface
+    - Alarms and warnings
+    - Security
+    - User interface
+    - Data definition and database
+    - Installation and acceptance
+    - Operation methods and maintenance
+    - Networking
+    - User maintenance
+    - Regulatory
+- Hit `Submit new issue` and theissue will be rendered like this:
+  ![](https://hackmd.io/_uploads/B1m9SkVZ6.png)
+
+  - Note that the `design-component-d` has been added as "Drug dosage" was added as a component type with a short code of `D`.
+
+- The first comment of this issue will be used by `qw` for tracking, but you can edit anything manually after the `Other information` header and this will not interfere with the tool
+- The parent issue will have been updated with this issue number
+  ![](https://hackmd.io/_uploads/HJODNgiya.png)
+
+### Design outputs and design verification
+
+- Create a pull request from github and follow the instructions on the template
+  ![](https://hackmd.io/_uploads/BkYpd7ikp.png)
+  - If the `qw-ignore` tag is added, then this PR does is not related to the medical device aspect of the software
+- In this example, the pull request contains the design outputs and design verification. These can be added seperately, where the design verification would also be linked to the design input.
+  ![](https://hackmd.io/_uploads/ryp39Xj1a.png)
+- QW will check that the chain of design items are able to be processed and fully signed off,
+  when this is successful the github action will pass
+  ![](https://hackmd.io/_uploads/H1Ix67i16.png)
+- QW requires a pull request that is labelled as a `design-validation` or `design-verification` to have at least one automated test that targets a specific `design-output`, `design-input` or `user-need`
+  - Tests are tracked manually in `.qw/test_mapping.csv` where multiple issues are separated by `;`, in the following structure
+    test name | issue(s) targetted
+    -- | --
+    InputVerifierTests::testWeightIsNegative | 40
+    InputVerifierTests::testConversionOfWeightUnits | 4;23
+    acceptance_test_user_entry.docx | 2
+  - The development team may want to write a script that allows them to get the names of all tests run and ensure that automated tests listed in the csv still exist, bonus points if you make this an automated test that fails if a test doesn't exist
+  - Adding the manual verification test scripts to version control is ideal, but you may also list the name
+  - the QW CI task will can fail if:
+    - No parent issue is given for a PR
+    - A test doesn't exist which is mapped to the parent issue
+    - The structure of the `test_mapping.csv` has been altered, or `;` has not been used to separate targetted issues
+    - A test name is duplicated in `test_mapping.csv`
+
+### Design validation
+
+- QW does not require design validation documentation to be added to the repository,
+  but you may add the test script.
```

Comment:
I'm not sure if we're talking across eachother. This was ment to be the proof of validation work, e.g. validation of a user need, so a test script. rather than generating documentation

Reply:
I'm more talking about the first clause in that statement. Shouldn't we be aiming to produce validation documentation? Not sure. What are we envisioning for validation documentation?